std_msgs/Header header
float64 r_reader             # r_reader reading
float64 r_reader_variance    # r_reader_variance reading


